charts/rancher-vsphere-cpi/rancher-vsphere-cpi/100.0.0/templates/vsphere-cpi-rbac.yaml

```diff
@@ -105,23 +106,24 @@ roleRef:
 subjects:
 - apiGroup: ""
   kind: ServiceAccount
-  name: cloud-controller-manager
+  name: vsphere-cloud-controller-manager
   namespace: {{ .Release.Namespace }}
 - apiGroup: ""
   kind: User
-  name: cloud-controller-manager
+  name: vsphere-cloud-controller-manager
```

Comment:
> Where is the `User` CR from? I do not find its definition in the chart.
@jiaqiluo  These are all changes from upstream that are already present in the chart and I only changed the name to avoid conflicts with rke2. However, is this here what you are referring to?

Reply:
I just found that they are defined in another chart: `rancher-vsphere-csi` 😆

Reply:
Ah, that's what you meant. Yeah, cpi and csi go together 😄